docs/sources/developer/Profiling.md

```diff
@@ -3,30 +3,44 @@ tuning.
 
 ## Profiling CPU usage
 
-### Profiling parsers
+The CPU usage of various parts of Plaso can processes can be profiled
```

Comment:
something is wrong here, remove "can processes" ?

Reply:
ack, thx for flagging, reworded